### PR TITLE
Add 2 Chains

### PR DESCRIPTION
**Pull request by nodeiistt**

### CODE REVIEW

File: chains/mainnet/realio.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    "chain_name": "realio",
+    "alias": "realio",
+    "api": "https://rest.cosmos.directory/realio",
+    "rpc": ["https://rpc.cosmos.directory/realio"],
+    "snapshot_provider": "",
+    "addr_prefix": "realio",
+    "coin_type": "494",
+    "min_tx_fee": "8000",
+    "assets": [{
+        "base": "ario,arst",
```
Comment (liangping) [inline chains/mainnet/realio.json:11]: Don't support "," for seperator of two base denoms, add two assets if you need

File: chains/mainnet/sge.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
+{
+    "chain_name": "sge",
+    "alias": "sge",
+    "api": "https://rest.cosmos.directory/sge",
+    "rpc": ["https://rpc.cosmos.directory/sge"],
```
Comment (liangping) [inline chains/mainnet/sge.json:5]: Is it possible to add more endpoints?

**Review by liangping — CHANGES_REQUESTED** [review]

**Comment by nodeiistt** [comment]
done

**Review by liangping — CHANGES_REQUESTED** [review]

File: chains/mainnet/realio.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
+{
+    "chain_name": "realio",
+    "alias": "realio",
+    "api": "https://rest.cosmos.directory/realio",
+    "rpc": ["https://rpc.cosmos.directory/realio"],
```
[the remark this answers is not in the document]
Comment (nodeiistt) [reply chains/mainnet/realio.json]: > please add more endpoint here. and please submit prettify the json.

done!

**Review by liangping — APPROVED** [review]

**Pull request merged**